deployment/digital-ocean/https/README.md

```diff
@@ -1,15 +1,16 @@
 # Setup Ingress and HTTPS
 
 Follow [this quick start guide](https://docs.cert-manager.io/en/latest/tutorials/acme/quick-start/index.html) and install certmanager via helm and tiller:
+[This resource was also helpful](https://docs.cert-manager.io/en/latest/getting-started/install/kubernetes.html#installing-with-helm)
 
-```text
+```bash
 $ kubectl create serviceaccount tiller --namespace=kube-system
 $ kubectl create clusterrolebinding tiller-admin --serviceaccount=kube-system:tiller --clusterrole=cluster-admin
 $ helm init --service-account=tiller
+$ helm repo add jetstack https://charts.jetstack.io
 $ helm repo update
-$ helm install stable/nginx-ingress
-$ kubectl apply -f https://raw.githubusercontent.com/jetstack/cert-manager/release-0.6/deploy/manifests/00-crds.yaml
-$ helm install --name cert-manager --namespace cert-manager stable/cert-manager
+$ kubectl apply -f https://raw.githubusercontent.com/jetstack/cert-manager/release-0.11/deploy/manifests/00-crds.yaml
+$ helm install --name cert-manager --namespace cert-manager --version v0.11.0 jetstack/cert-manager
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-11-11T15:31:22Z" title="Monday, November 11th 2019, 4:31:22 pm +01:00">Nov 11, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,15 +1,16 @@
 # Setup Ingress and HTTPS
 
 Follow [this quick start guide](https://docs.cert-manager.io/en/latest/tutorials/acme/quick-start/index.html) and install certmanager via helm and tiller:
+[This resource was also helpful](https://docs.cert-manager.io/en/latest/getting-started/install/kubernetes.html#installing-with-helm)
 
-```text
+```bash
 $ kubectl create serviceaccount tiller --namespace=kube-system
 $ kubectl create clusterrolebinding tiller-admin --serviceaccount=kube-system:tiller --clusterrole=cluster-admin
 $ helm init --service-account=tiller
+$ helm repo add jetstack https://charts.jetstack.io
 $ helm repo update
-$ helm install stable/nginx-ingress
-$ kubectl apply -f https://raw.githubusercontent.com/jetstack/cert-manager/release-0.6/deploy/manifests/00-crds.yaml
-$ helm install --name cert-manager --namespace cert-manager stable/cert-manager
+$ kubectl apply -f https://raw.githubusercontent.com/jetstack/cert-manager/release-0.11/deploy/manifests/00-crds.yaml
+$ helm install --name cert-manager --namespace cert-manager --version v0.11.0 jetstack/cert-manager
```


I cannot tell whether this works or not, but if it worked for https://develop.human-connection.org/ I'm pretty sure it will work for us, too!